login.js

```diff
@@ -0,0 +1,75 @@
+const bcrypt = require('bcrypt');
+const { app, databaseConnection, con } = require('./app');
```

Comment:
Why import databaseConnection and con from app.js ? You can import it from databaseConnection.js directly